common/src/main/resources/expiration.properties.example

```diff
@@ -0,0 +1,3 @@
+seconds_verification_email_valid=86400
+days_refresh_expiration=604800000
```

Comment:
I don't think we're keeping refresh tokens for 604 million days

Reply:
oh no

Reply:
oh yes